### DISCUSSION TITLE
How  to get badge in holopin

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/Stackoverflow-Analysis#374] [Q&A] [answered]
Give me some guidance to get holopin badge also github bagde.

**Comment by sanjay-kv** [comment]
recommend you to watch this playlist: https://www.youtube.com/watch?v=sbyXpflAXkQ&list=PLrLTYhoDFx-kiuFiGQqVpYYZ56pIhUW63&index=10

**Reply by Hassassin-10** [reply, marked as answer]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.

**Comment by muhammadashraf23** [comment]
> Give me some guidance to get holopin badge also github bagde.

To get Holopin badges:

Participate in open-source events like GSSoC or Hacktoberfest.
Join events hosted on Holopin for exclusive badges.
Stay active in developer communities that distribute badges.

For GitHub badges:
Contribute to open-source projects (e.g., Hacktoberfest).
Apply for special programs like GitHub Campus Expert.
Earn badges through GitHub achievements (e.g., Arctic Code Vault, Open Source Contributor).

**Reply by Hassassin-10** [reply]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.

**Comment by PauLopNun** [comment]
Also when u get the badges u can add this to your markdown file   [! [An image of @yourusername Holopin badges, which is a link to view their full Holopin profile](https://holopin.me/@yourusername)](https://holopin.io/@yourusername

**Reply by Hassassin-10** [reply]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.

**Comment by Swasthiga-0618** [comment]
> Give me some guidance to get holopin badge also github bagde.

Hey ! you can get all the badges related to open source contribution specifically in hacktoberfest is by contributing to new beginner friendly projects that are assigned by the maintainers and after contributing in either of code or non code part ,you have to wait for 7 days PR review after you will get holopin badges .(note):1.Find projects that you want to contribute 2.Go through guidelines of contributing for particular project 3.Label it properly

**Reply by Hassassin-10** [reply]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.

**Comment by Prince-Tirkey** [comment]
You need to participate in Hactoberfest.
For registering you get 1 Hactoberfest registration holopin badge
and make 4 PR's to get level 1, level 2, 3, 4 badges , only if your PR got accepted

**Reply by Hassassin-10** [reply]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.

**Comment by Hassassin-10** [comment]
Participate in an Event or Challenge: Many badges are awarded for participating in events like Hacktoberfest or other coding challenges.

Create an Account: Visit the Holopin website and sign up for an account.

Claim Your Badge: Once you've participated in an event or completed a challenge, you'll receive a link to claim your badge.

Add to Your Profile: Copy the badge URL and add it to your GitHub, LinkedIn, or other social media profiles to showcase your achievement.
